src/guide/android_troubleshooting.md

```diff
@@ -0,0 +1,76 @@
+# Troubleshooting Android
+
+When including the Breez SDK and/or Notification Plugin into your application you might come across issues compiling because: 
+- it uses `kotlinx-serialization`
+- it relies on JNA library
+
+If you do, there are several steps you can take to compile and build properly, even if they do not using the Notification Plugin feature.
+
+## kotlinx-serialization
+
+Starting with this 1.8.20 release, the Kotlin K2 compiler has a preview version of the serialization plugin. It's recommended for users to upgrade their Kotlin version to at least 1.8.20 or [set up the serialization plugin](https://github.com/Kotlin/kotlinx.serialization?tab=readme-ov-file#setup) on their projects explicitly with older versions.
+
+## JNA library
```

Comment:
JNA library code obfuscation issue may not be apparent until users create a _`"release"`_ mode of their app when with **maximum optimization and minimal footprint size** that'll be used when deploying their app and I think we should add a warning about this in this section.